components/mpas-seaice/cime_config/buildnml

```diff
@@ -245,8 +245,8 @@ def buildnml(case, caseroot, compname):
     elif ice_grid == 'ECwISC30to60E2r1':
         grid_date = '210413'
         grid_prefix = 'seaice.ECwISC30to60E2r1'
-        decomp_date = '200916'
-        decomp_prefix = 'mpas-seaice.graph.info.'
+        decomp_date = '230314'
+        decomp_prefix = 'partition/mpas-seaice.graph.info.'
```

Comment:
@darincomeau, does that look right this time?

Reply:
looks right - thanks for the quick fix! I'm testing each grid in part to make sure the new `partitions` directories created on the *other* LCRC inputdata directory have the right permissions.

Reply:
oh actually, just noticed this one is `partition` whereas the others are `partitions`

Reply:
Oh, no!

Reply:
Fixes again, hopefully for the last time 🤞🏻